docs/src/examples/components/Animation/Types/AnimationExampleDirection.tsx

```diff
@@ -21,18 +21,26 @@ const AnimationExampleDirection = () => (
       <Text content="Reverse" />
       <Text content="Alternate" />
       <Text content="Alternate reverse" />
-      <Animation name="spinner">
-        <Icon name="mention" circular bordered />
-      </Animation>
-      <Animation name="spinner" direction="reverse">
-        <Icon name="mention" circular bordered />
-      </Animation>
-      <Animation name="spinner" direction="alternate">
-        <Icon name="mention" circular bordered />
-      </Animation>
-      <Animation name="spinner" direction="alternate-reverse">
-        <Icon name="mention" circular bordered />
-      </Animation>
+      <div>
```

Comment:
This change looks strange...

Reply:
Previously the Animation component was creating a div, so the styles from the Grid were applied on it. Now those are applied on the icons and it is breaking the animation. If the Icon is used outside of Grid, there is no need for additional div.